sdk/lib/stdio/xmake.lua

```diff
@@ -2,3 +2,15 @@ library("stdio")
   add_deps("string")
   set_default(false)
   add_files("printf.cc")
+  add_options("debug-print-doubles")
```

Comment:
Given its use in `lib/stdio`, perhaps the option just should be `print-doubles` (and similarly for `float`)?